Common/include/option_structure.hpp

```diff
@@ -887,23 +887,23 @@ static const MapType<std::string, ENUM_MATRIX_COLORING> MatrixColoring_Map = {
 /*!
  * \brief Types of slope limiters
  */
-enum ENUM_LIMITER {
-  NO_LIMITER           = 0, /*!< \brief No limiter. */
-  VENKATAKRISHNAN      = 1, /*!< \brief Slope limiter using Venkatakrisnan method (stencil formulation). */
-  VENKATAKRISHNAN_WANG = 2, /*!< \brief Slope limiter using Venkatakrisnan method, eps based on solution (stencil formulation). */
-  BARTH_JESPERSEN      = 3, /*!< \brief Slope limiter using Barth-Jespersen method (stencil formulation). */
-  VAN_ALBADA_EDGE      = 4, /*!< \brief Slope limiter using Van Albada method (edge formulation). */
-  SHARP_EDGES          = 5, /*!< \brief Slope limiter using sharp edges. */
-  WALL_DISTANCE        = 6  /*!< \brief Slope limiter using wall distance. */
+enum class ENUM_LIMITER {
```

Comment:
Remove the ENUM_ prefix please, to keep things shorter.
Sounds like some time of undefined behaviour, have you tried running valgrind?
The AdjLimiter are only for continuous adjoints, one for species is not needed.

Reply:
thanks for the comment. Haven't tried valgrind yet, I'm not looking forward to doing parallel debugging. But it looks like it might be necessary.

Reply:
If it is due to an uninitialized variable it will show up also in serial.